### ISSUE TITLE
some question about the post process

### ISSUE THREAD

**Issue by LLLLincoln** [isds-neu/PhyCRNet#20]
Hello author,
  I read several issues you responded to in the other question, then I ran the code. 
  First, I deleted "load_checkpoint" in the train function. then I set time_batch_size=100, time_steps=101, and n_iters_adam=500, I trained the network, I found that there was a huge loss, and also in the post_process function it came an error "'c' argument has 16641 elements, which is inconsistent with 'x' and 'y' with size 16384.
  Can you help me see if there are some problems with the code running steps! Thank you very much!

**Comment by biterliu** [comment]
change function post_process as:
 u_pred = output[num, 0, 1:-2, 1:-2].detach().cpu().numpy()
